### ISSUE TITLE
Rule Request: implicit_return should be configurable for computed variables vs functions

### ISSUE THREAD

**Issue by mozeryansky** [realm/SwiftLint#3337]
### New Issue Checklist

- [X] Updated SwiftLint to the latest version
- [X] I searched for [existing GitHub issues](https://github.com/realm/SwiftLint/issues)

### New rule request

Can implicit_return have a configuration to only work for computed variables and not also functions.

```
var x: Int {
    return 0 // violation
}
var y: Int {
    0 // OK
}
function makeX() -> Int {
    return 0 //  OK
}
```

It's not instantly obvious to know if a function is performing an action or returns a value whereas a computed variable always is returning a value.

**Comment by fedeci** [comment]
You can already configure `implicit_return` in this way:

```yml
...
opt_in_rules:
  - implicit_return
...
implicit_return:
  included:
    - getter
    - function
    - closure
...
```

**Comment by mozeryansky** [comment]
Oh! Thanks
